sdk/test/Plugin/Haskell/Chart1.dump-splices

```diff
@@ -0,0 +1,171 @@
+/Users/axel-axul/Developer/statecharts/sdk/test/Plugin/Haskell/Chart1.hs:15:7-75: Splicing expression
```

Comment:
not quite sure what these dump splice files are. Should they be ignored?

Reply:
yes, ignore